controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -327,3 +351,76 @@ func (r *OCSInitializationReconciler) getEnableNFSKeyValue() string {
 
 	return "false"
 }
+func (r *OCSInitializationReconciler) reconcileUXBackendSecret(initialData *ocsv1.OCSInitialization) error {
+
+	var err error
+
+	secret := &corev1.Secret{}
+	secret.Name = "ux-proxy-secret"
+	secret.Namespace = initialData.Namespace
+
+	opResult, err := ctrl.CreateOrUpdate(r.ctx, r.Client, secret, func() error {
+
+		if err := ctrl.SetControllerReference(initialData, secret, r.Scheme); err != nil {
+			return err
+		}
+		secret.StringData = map[string]string{"session_secret": strings.Replace(uuid.New().String()+uuid.New().String(), "-", "", -1)}
```

Comment:
`cookie-secret string: the seed string for secure cookies` as per the docs
since it's a seed string I assume having a randomly generated string should be enough,

Reply:
> having a randomly generated string should be enough
- better to add the lifetime as well, like should it be per reconcile or per deployment